.orchestra/config/components/vscode-web.yml

```diff
@@ -0,0 +1,62 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+
+#@yaml/text-templated-strings
+---
+#@ def vscode_web_component():
+repository: vscode
+license: LICENSE.txt
+default_build: default
+binary_archives: private
+builds:
+  #@ for flavor in ["default", "debug"]:
+  (@= flavor @):
+    dependencies:
+    - revng
+    build_dependencies:
+    - revng-js-prerequisites
+    - revng-ui
+    - revng-theme
+    configure: |
+      rm -rf "$BUILD_DIR/vscode"
+      mkdir -p "$BUILD_DIR/vscode"
+      cd "$BUILD_DIR/vscode"
+
+      rsync -aq \
+        --exclude=.git \
+        --exclude-from=<(git -C "$SOURCE_DIR" ls-files --exclude-standard -oi --directory) \
+        "$SOURCE_DIR/" .
+
+      find . -type f -name yarn.lock -printf '%h\n' | \
+      while IFS= read -r directory; do
+        (cd "$directory"; "$ORCHESTRA_DOTDIR"/support/js-populate-cache yarn)
+      done
+    install: |
+      cd "$BUILD_DIR/vscode"
+      rsync -aq \
+        --exclude-from=<(git -C "$SOURCE_DIR" ls-files --exclude-standard -oi --directory) \
+        "$SOURCE_DIR/" .
+
+      COMMANDS_DIR="${DESTDIR}${ORCHESTRA_ROOT}/lib/python/revng/cli/_commands"
+      mkdir -p "$COMMANDS_DIR"
+      cp -a revng/util/launch-vscode-web.py "$COMMANDS_DIR"
+
+      ELECTRON_SKIP_BINARY_DOWNLOAD=1 \
+      PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1 \
+        yarn
```

Comment:
Add a comment stating what still requires internet access to best of our current knowledge.